src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothLeAdapterImpl.cs

```diff
@@ -132,14 +132,10 @@ internal void RegisterGattConnectionStateChangedEvent()
             _gattConnectionStateChangedCallback = (int result, bool connected,
                                     string remoteDeviceAddress, IntPtr userData) =>
             {
-                if (_gattConnectionStateChanged != null)
-                {
-                    Log.Info(Globals.LogTag, "Setting gatt connection state changed callback" );
-                    GattConnectionStateChangedEventArgs e = new GattConnectionStateChangedEventArgs (result,
+                Log.Info(Globals.LogTag, "Setting gatt connection state changed callback");
+                GattConnectionStateChangedEventArgs e = new GattConnectionStateChangedEventArgs(result,
                         connected, remoteDeviceAddress);
-
-                    _gattConnectionStateChanged(null, e);
-                }
+                _gattConnectionStateChanged?.Invoke(null, e);
```

Comment:
Usually, sender on non-static event should be this object, Anyway, I don't care it, because actually not used.

Reply:
Thank you for comment. I will use 'this' object on sender in later.